src/react/databrowser/buckets/details/Overview.tsx

```diff
@@ -192,25 +195,38 @@ function Overview({ bucket, ingestionStates }: Props) {
                 <T.Value>
                   {bucketInfo.objectLockConfiguration.ObjectLockEnabled ===
                     'Disabled' && (
-                    <Toggle
-                      disabled={loading}
-                      toggle={bucketInfo.isVersioning}
-                      label={
-                        bucketInfo.versioning === 'Enabled'
-                          ? 'Active'
-                          : bucketInfo.versioning === 'Disabled'
-                          ? 'Inactive'
-                          : bucketInfo.versioning
-                      }
-                      onChange={() =>
-                        dispatch(
-                          toggleBucketVersioning(
-                            bucket.name,
-                            !bucketInfo.isVersioning,
-                          ),
+                    <Tooltip
+                      overlay={
+                        isBucketHostedOnAzureOrGCP ? (
+                          <>
+                            Enabling versioning is not possible due to the
+                            bucket being hosted on Azure or Google Cloud.
```

Comment:
Nit: We can probably say on azure for azure hosted buckets and on Google Cloud for the google hosted ones